frontend/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://localhost:3001",
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.15.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
```

Comment:
This package.json is currently missing `react-router-dom`, which is required for any routing in your project. To fix, just run `npm i react-router-dom@5` in the frontend directory, then commit the new package.json to your repo. I'd recommend making this change now in case you ever wanted anyone to clone this repo in the future.